app/assets/v2/js/vue-components.js

```diff
@@ -9,10 +9,16 @@ Vue.mixin({
     };
   },
   methods: {
-    chatWindow: function(handle) {
+    chatWindow: function(channel, dm) {
+      dm = dm | !channel;
+      channel = channel | 'town-square';
```

Comment:
are you sure you want to use one pipe here? is not suppose to be the logical operator `||` ?

Reply:
https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators#Bitwise_OR

Reply:
ah yes is typo should be `||`